src/main/java/com/example/h2/common/PassBasedEnc.java

```diff
@@ -0,0 +1,46 @@
+package com.example.h2.common;
+
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.Random;
+
+public class PassBasedEnc {
+    /* Declaration of variables */
+    private static final Random random = new SecureRandom();
+    private static final String characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```

Comment:
Remove unused variable. If you are not using them in your code

Reply:
okk